tools/nl/embeddings/data/preindex/sdg/duplicate_names.csv

```diff
@@ -1,24 +1,58 @@
 dc/topic/sdg_1.5.3,dc/topic/sdg_11.b.1,Number of countries that adopt and implement national disaster risk reduction strategies in line with the Sendai Framework for Disaster Risk Reduction 2015-2030
 dc/topic/sdg_1.5.4,dc/topic/sdg_11.b.2,Proportion of local governments that adopt and implement local disaster risk reduction strategies in line with national disaster risk reduction strategies
+dc/topic/sdg_12,dc/topic/sdg_1.2,sdg 12
+dc/topic/sdg_12,dc/topic/sdg_1.2,sdg12
+dc/topic/sdg_12.1,dc/topic/sdg_1.2.1,sdg 121
+dc/topic/sdg_12.1,dc/topic/sdg_1.2.1,sdg121
+dc/topic/sdg_12.2,dc/topic/sdg_1.2.2,sdg 122
+dc/topic/sdg_12.2,dc/topic/sdg_1.2.2,sdg122
 dc/topic/sdg_8.4.1,dc/topic/sdg_12.2.1,"Material footprint, material footprint per capita, and material footprint per GDP"
 dc/topic/sdg_8.4.2,dc/topic/sdg_12.2.2,"Domestic material consumption, domestic material consumption per capita, and domestic material consumption per GDP"
 dc/topic/sdg_4.7.1,dc/topic/sdg_12.8.1,(b) curricula
 dc/topic/sdg_4.7.1,dc/topic/sdg_12.8.1,(c) teacher education
 dc/topic/sdg_4.7.1,dc/topic/sdg_12.8.1,Extent to which (i) global citizenship education and (ii) education for sustainable development are mainstreamed in (a) national education policies
 dc/topic/sdg_4.7.1,dc/topic/sdg_12.8.1,and (d) student assessment
 dc/topic/sdg_7.b.1,dc/topic/sdg_12.a.1,Installed renewable energy-generating capacity in developing countries (in watts per capita)
+dc/topic/sdg_13,dc/topic/sdg_1.3,sdg 13
+dc/topic/sdg_13,dc/topic/sdg_1.3,sdg13
+dc/topic/sdg_13.1,dc/topic/sdg_1.3.1,sdg 131
+dc/topic/sdg_13.1,dc/topic/sdg_1.3.1,sdg131
 dc/topic/sdg_1.5.1,dc/topic/sdg_13.1.1,"Number of deaths, missing persons and directly affected persons attributed to disasters per 100K population"
 dc/topic/sdg_1.5.3,dc/topic/sdg_13.1.2,Number of countries that adopt and implement national disaster risk reduction strategies in line with the Sendai Framework for Disaster Risk Reduction 2015-2030
 dc/topic/sdg_1.5.4,dc/topic/sdg_13.1.3,Proportion of local governments that adopt and implement local disaster risk reduction strategies in line with national disaster risk reduction strategies
 dc/topic/sdg_4.7.1,dc/topic/sdg_13.3.1,(b) curricula
 dc/topic/sdg_4.7.1,dc/topic/sdg_13.3.1,(c) teacher education
 dc/topic/sdg_4.7.1,dc/topic/sdg_13.3.1,Extent to which (i) global citizenship education and (ii) education for sustainable development are mainstreamed in (a) national education policies
 dc/topic/sdg_4.7.1,dc/topic/sdg_13.3.1,and (d) student assessment
+dc/topic/sdg_14,dc/topic/sdg_1.4,sdg 14
+dc/topic/sdg_14,dc/topic/sdg_1.4,sdg14
+dc/topic/sdg_14.1,dc/topic/sdg_1.4.1,sdg 141
+dc/topic/sdg_14.1,dc/topic/sdg_1.4.1,sdg141
+dc/topic/sdg_14.2,dc/topic/sdg_1.4.2,sdg 142
+dc/topic/sdg_14.2,dc/topic/sdg_1.4.2,sdg142
+dc/topic/sdg_15,dc/topic/sdg_1.5,sdg 15
+dc/topic/sdg_15,dc/topic/sdg_1.5,sdg15
+dc/topic/sdg_15.1,dc/topic/sdg_1.5.1,sdg 151

```

Comment:
I have no fixed this so "sdg 1.5.1" will not match to "sdg 15.1" but if someone types "sdg 151" it will match to sdg 15.1 and not to sg 1.5.1 which seems Ok.